Source/Demos/Demo.PrimitiveBatch/Game1.cs

```diff
@@ -0,0 +1,102 @@
+﻿using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using Microsoft.Xna.Framework.Input;
+using MonoGame.Extended.Graphics;
+using MonoGame.Extended.Graphics.Batching;
+using SpriteBatch = MonoGame.Extended.Graphics.Batching.SpriteBatch;
```

Comment:
`Vector3` as position (depth is no longer a parameter); I will add ability to pass 1 or 4 `Color` structs to use for the 4 corners of the quad; consistent API to `PrimitiveBatch` for choosing a `BatchDrawStrategy`, maximum batch size, and using a `IDrawContext` with a `BatchSortMode` for switching and batching by state (blending, sampling, textures, depth stencil, rasterization) / effects (vertex shaders, pixel shaders).